content/source/docs/extend/best-practices/versioning.html.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: "extend"
+page_title: "Extending Terraform: Versioning Best Practices"
+sidebar_current: "docs-extend-best-practices-versioning"
+description: |-
+  Recommendations for version numbering and documentation.
+---
+
+# Versioning
+
+Given the breadth of available Terraform providers, ensuring a consistent experience means setting a standard guideline for compatibility promises. These guidelines are enforced for providers released by HashiCorp and are recommended for all community providers.
+
+## Table of Contents
```

Comment:
the page renders short enough if you drop the table of contents I think it flows pretty well? Your call there, I don't know if we have a standard practice to include a table of contents.

Reply:
ToC removed. Premature optimization. 😉